### DISCUSSION TITLE
Images showing up outside of border how to change the position ?

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [sxyazi/yazi#464] [1. Q&A] [answered]
![image](https://github.com/sxyazi/yazi/assets/127573781/b33ed8b4-6488-4aae-945e-eb85351c7c5b)

**Comment by sxyazi** [comment]
Hi, what's your Yazi version? and what's the terminal you're running Yazi in?

**Comment by sachinsenal0x64** [comment]
Version = yazi 0.1.5
Terminal = kitty

**Comment by sxyazi** [comment]
I noticed you're using full borders, did you [modify `preview_offset` to the value in that tip](https://yazi-rs.github.io/docs/tips#full-border)?

**Reply by sachinsenal0x64** [reply]
Yes but nothing happen 

![image](https://github.com/sxyazi/yazi/assets/127573781/2311843c-53f2-49fe-a4dd-7170ca03d37c)

**Reply by sxyazi** [reply, marked as answer]
It's `theme.toml` not `yazi.toml`

**Reply by sachinsenal0x64** [reply]
Now all working thanks btw can i change position ?

![image](https://github.com/sxyazi/yazi/assets/127573781/ea3c60c5-0b32-4ade-977e-b677c388d4d3)

**Reply by sxyazi** [reply]
It will be supported in Yazi v0.2.0 [in a custom previewer manner](https://github.com/sxyazi/yazi/pull/401). I will release it next week, some preparation is needed as it's a breaking change version.
